app/models/order.rb

```diff
@@ -1,7 +1,7 @@
 class Order < ApplicationRecord
   include OwnerField
   include Discard::Model
-  include Catalog::DiscardRestore
+  include V1x0::Catalog::DiscardRestore
```

Comment:
@syncrou Wouldn't the models also have to be versioned at some point?

Reply:
I will probably make a change here so we're not using a versioned `DiscardRestore`. But on the conversation of versioning the models. That is a very large conversation. We would potentially need to manage multiple dbs or totally separate apps. 

For now I think we need to leave that conversation for later.